testing/dart/observatory/skp_test.dart

```diff
@@ -35,14 +35,14 @@ void main() {
       builder.addPicture(Offset.zero, picture);
       final Scene scene = builder.build();
 
-      window.render(scene);
+      PlatformDispatcher.instance.implicitView!.render(scene);
```

Comment:
The comments on line 40 and lines 42-43 look like test code that was never removed. Consider deleting it.

Reply:
Will do!